src/example-types/legacy/smartIntervalHistogram.js

```diff
@@ -1,12 +1,12 @@
-let _ = require('lodash/fp')
-let statsResults = require('./statistical').result
-let calcSmartInterval = require('../../utils/smartInterval').calcSmartInterval
+import _ from 'lodash/fp.js'
+import statsResults from './statistical.js'
```

Comment:
small nit: this import is no longer `statsResults` but is really just `stats` or `statistical`